### ISSUE TITLE
Time and space

### ISSUE THREAD

**Issue by matt439miller** [jakesower/rpg-2024#1]
Need some time and space rules. How far can you move per round? Which keys into how long a round is, and how much stuff you can do. I'd personally just reuse everything from middle, as we already did a lot of work on that, and it's well tested for action speeds and move rates.

**Comment by jakesower** [comment]
- Keep the combat clock with 12 segments and an "end of round" spot between the 12 and 1. I'd like to add something so that whoever moves their token to a segment first, goes last when that segment is reached (a stack). That way you're not relying on initiative for breaking ties, and faster actions feel a little faster.
- I added a mechanic for `stunned` and `dazed` which helps avoid moving tokens around and avoids the "more than 12" issue
- Movement takes (1 + 2*spaces) time. So move 1 takes 3, move 2 takes 5, etc.

**Comment by matt439miller** [comment]
I like the stack rule, but it's a little board gamey. Implies the existence of a physical object. Good rule, but needs a different implementation.  "Whomever acted last acts next". Will have some weird tactical effect, where you take a long action to 'stack' an enemy and then act again. Actually, a very nice solution to the whole 'charge' problem, as well as attacks of opportunity. If you time it right, you CAN move through a threatened square. Does raise the issue of a readied action... use the 'waiting off the wheel' mechanic again?

**Comment by jakesower** [comment]
Yeah. Off the clock waiting would need to be in there for sure. I think physical elements to a game improve them and so I'm not necessarily opposed to keeping some "board gamey" aspects.

**Comment by matt439miller** [comment]
That moves into a different space. Demote hero quest. Do you know ow gloom haven?

**Comment by jakesower** [comment]
I do not own Gloomhaven. Yeah, I suppose the point of a pencil and paper RPG is you can make all the game components with pencil and paper without too much trouble. Combat clock and whatnot.
